cpmpy/solvers/ortools.py

```diff
@@ -217,7 +217,9 @@ def solve(self, time_limit=None, assumptions=None, solution_callback=None, **kwa
 
             # translate objective
             if self.has_objective():
-                self.objective_value_ = self.ort_solver.ObjectiveValue()
+                ort_obj_val = self.ort_solver.ObjectiveValue()
+                assert int(ort_obj_val) == ort_obj_val, "Objective value should be integer, please report on github"
```

Comment:
Does ortools return numpy ints? Do we need this for other solvers as well?

Reply:
No, it returns floats (but always `.0` of course as everything is int)

Reply:
I checked for the other solvers, they all return `int` except for Gurobi, which is also patched now